### PR TITLE
Fix List to produce items instead of objects

### PR DESCRIPTION
**Pull request by transamericamoon**
From what i've seen List contains items and templates contain objects.

### CODE REVIEW

**Comment by jcantrill** [comment]
@transamericamoon I am unable to find the reason why 'objects' was introduced.  The only thing I can say/recall is that it was introduced during a refactoring in which something was believed to be true and apparently was not.  Looking at the api for List, I agree that I do not see any reference to 'object'.  Could you add https://github.com/openshift/openshift-restclient-java/pull/216/files#diff-62e85ab15c1ef21309b9638e77e8790cR44 to your fix and squash your commits?

**Comment by jcantrill** [comment]
This will fix #215

**Comment by jeffmaury** [comment]
Unit tests ?

**Comment by jcantrill** [comment]
- ListTest

 already https://github.com/openshift/openshift-restclient-java/blob/master/src/test/java/com/openshift/internal/restclient/model/v1/ListTest.java  
- please see note regarding also removing Objects from: https://github.com/openshift/openshift-restclient-java/pull/216/files#diff-62e85ab15c1ef21309b9638e77e8790cR44
- please squash your commits

**Comment by jeffmaury** [comment]
Yes but the test doesn't test the fixed feature/behavior

**Comment by transamericamoon** [comment]
@jeffmaury Not sure how to test it since i don't have access to the ModelNode in the test.  I serialize and check for "items" previously it was creating "objects".

File: src/main/java/com/openshift/internal/restclient/model/List.java [hunk 1]
```diff
@@ -41,10 +41,9 @@ public List(ModelNode node, IClient client, Map<String, String []> propertyKeys)
 	@Override
 	public Collection<IResource> getItems(){
 		if(items == null) {
-			String key = getNode().has(OBJECTS) ? OBJECTS : ITEMS;
-			ModelNode listNode = get(key);
+			ModelNode listNode = get(ITEMS);
 			if (listNode.isDefined()) {
-				Collection<ModelNode> nodes = get(key).asList();
+				Collection<ModelNode> nodes = get(ITEMS).asList();
```
Comment (jcantrill) [inline src/main/java/com/openshift/internal/restclient/model/List.java]: listNode.asList()

**Comment by jcantrill** [comment]
@transamericamoon the only way to get access to the node is via casting to a KubernetesResource and calling the getNode methods.

**Comment by transamericamoon** [comment]
Made the suggested changes.  Thanks for the guidance.

**Comment by transamericamoon** [comment]
I just realized I used spaces instead of tabs in the unit test, sorry :(

**Pull request closed without merging**